docs/superpowers/plans/2026-05-13-pd-console-ui-ux-refactoring.md

```
@@ -0,0 +1,1149 @@
+
+**Tech Stack:** React 19, Tailwind CSS v4, shadcn/ui (copy-paste), React Router v7, i18next, lucide-react, vitest + @testing-library/react
+
+**PRD:** Linear PRI-134
+
+---
+
+## File Structure Map
+
+### New Files
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)